### ISSUE TITLE
Determine and then add tags or some way to categorize

### ISSUE THREAD

**Issue by melodykramer** [18F/projects#23]
Thinking about how to tag our projects will be key to discovery mechanisms.

Two thoughts: the fewer categories, the better. @brittag smarted suggested grouping projects into three categories: **Learn and Use**, **Partner**, and **Contribute.**

“Contribute to 18F” might encompass micropurchase and Open Opportunities options (and the future bug bounty program) as well as contributing to our open source projects?

“Partner with 18F: learn about our projects with agency partners (both completed and in progress)”

“learn from 18F and use its materials”: material that's reuseable from other organizations.

I really like the idea of keeping this minimal, but would like @thisisdano's thoughts on this approach.

Alternatively or in addition, we could give people a dropdown list of tags from the blog: https://18f.gsa.gov/tags/ - This might also be a way to bring in related material from the blog. We would likely want to continually refresh this blog tag list, if new tags are added to the blog. (Not sure if this is the right approach or one we should take, just throwing it out there for preliminary discussion.) @gboone

**Comment by melodykramer** [comment]
Capturing this thought from @thisisdano as well: "Searching by Tag is an option, but there may be a solution closer to what we’re discussing here with curated lists? How is what we’re doing with projects also applicable to “ideas” / blog posts / docs?"

**Comment by gboone** [comment]
I definitely agree with keeping categories small and using the same tags the blog is using. A problem we have on the blog is that not many posts fall into any given tag, and we have dozens of tags. I think that's OK, tags are supposed to represent the breadth of the content. If we have other things (like projects) tagged similarly it opens up some interesting opportunities for cross referencing.

**Comment by melodykramer** [comment]
Yeah, I could see a nice embed on the blog with related projects...

**Comment by toolness** [comment]
This might be obvious, but one of my favorite design affordances here is the use of free-text auto-complete fields.  This allows folks to make up new tags if they really need to, but the autocomplete makes it really easy to converge on a single name rather than having a bunch of variations.

We could also have a cron job pull all the tags from the blog on a regular basis so that those pop up in the autocomplete too.
